scapy/layers/tls/keyexchange.py

```diff
@@ -154,7 +154,7 @@ class _TLSSignature(_GenericTLSSessionInheritance):
     #XXX 'sig_alg' should be set in __init__ depending on the context.
     """
     name = "TLS Digital Signature"
-    fields_desc = [SigAndHashAlgField("sig_alg", 0x0401, _tls_hash_sig),
+    fields_desc = [SigAndHashAlgField("sig_alg", 0x0804, _tls_hash_sig),
```

Comment:
Is this change needed?

Reply:
ping

Reply:
I think it's a safer default (that's what wikipedia says https://en.m.wikipedia.org/wiki/Probabilistic_signature_scheme ... romain did this).
previous was `sha256+rsa`, this is `sha256+rsaepss`.
I wonder if the "e" is a typo or not, since the 512 one has no e..